Training3/src/tasks/src/App.js

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import { useEffect, useState } from "react";
+import Modal from "./Modal";
+
+function App() {
+  const [isOpen, setIsOpen] = useState(false);
+  function handleClick() {
+    setIsOpen(!isOpen);
+  }
+  useEffect(() => {
+    if (isOpen) {
+      window.addEventListener("keypress", (e) => {
```

Comment:
@phuongluongZigvy we can't remove a function listener if you declare anymomus function like that, the function should declare as variable and pass this variable to the removeEventListener

Reply:
My solution: https://github.com/longhpzigvy/zv-react-training/pull/28/commits/43a523eac070e16bda50cb0b0cce9d03d1c79323#r715452922